packages/cli/lib/errorHandlers/apiErrors.js

```diff
@@ -257,6 +258,7 @@ function logApiErrorInstance(error, context) {
   // Use the new local-dev-lib error handler
   // NOTE: This will eventually replace the logic in logApiStatusCodeError
   if (error.isAxiosError) {
+    if (overrideErrors(error)) return;
```

Comment:
This doesn't need to be blocking if you don't agree, but this pattern seems a little confusing to me + `overrideErrors` is kind of a vague name and doesn't really say what it's actually doing.

Seeing that we do this for `statusCodeErrors`
```
 if (isApiStatusCodeError(error)) {
    logApiStatusCodeError(error, context);
    return;
  }
```

Would it make sense to just make a `isPlatformVersionError` function and then do
```
 if (isPlatformVersionError(error)) {
    logPlatformVersionError(error, context);
    return;
  }
```
where `logPlatformVersionError` is basically just the current `overrideErrors`

Reply:
Purposely chose this pattern, so we can reuse `overrideErrors` later, when we need to override normal error handlers to display custom errors for specific use cases. I'm planning to migrate missing scopes errors in projects to this pattern, and I think the sandboxes team has a few use cases, as well 👍